challengeutils/createchallenge.py

```diff
@@ -219,6 +222,9 @@ def createchallenge(syn, challenge_name, live_site=None):
     syn.setPermissions(project_staging, team_admin_id, admin_perms)
     syn.setPermissions(project_live, team_admin_id, admin_perms)
 
+    org_perms = ['CREATE', 'READ', 'UPDATE', 'DELETE', 'DOWNLOAD', 'MODERATE']
```

Comment:
Please see https://github.com/Sage-Bionetworks/challengeutils/blob/master/challengeutils/permissions.py

Please see permissions here.  You will want to give the organizers EDIT on the staging site and DOWNLOAD on the live site